packages/cli/src/commands/split/__tests__/index.test.ts

```diff
@@ -1,32 +1,50 @@
-import { iteratePaths } from '../index';
-import * as path from 'path';
-import * as openapiCore from '@redocly/openapi-core';
+// import { iteratePathItems } from '../index';
```

Comment:
We should remove commented out code, right?

Reply:
Or you are planning to fix the test, I guess...

Reply:
Yes, I will be working on those today. Thanks.